Index: mysql-5.5/mysql-test/t/disabled.def
===================================================================
--- mysql-5.5.orig/mysql-test/t/disabled.def	2012-03-06 13:10:09.085143548 -0800
+++ mysql-5.5/mysql-test/t/disabled.def	2012-03-06 15:23:22.905149613 -0800
@@ -15,3 +15,5 @@
 archive-big              : Bug#11817185 2011-03-10 Anitha Disabled since this leads to timeout on Solaris Sparc
 log_tables-big           : Bug#11756699 2010-11-15 mattiasj report already exists
 mysql_embedded           : Bug#12561297 2011-05-14 Anitha Dependent on PB2 changes - eventum#41836
+partition_rename_longfilename : Fails when building with sbuild and schroots
+file_contents            : Fails without bzr revision id
