=== modified file 'client/CMakeLists.txt'
--- a/client/CMakeLists.txt	2012-02-15 07:57:46 +0000
+++ b/client/CMakeLists.txt	2012-02-15 08:10:31 +0000
@@ -27,45 +27,52 @@
 
 ADD_DEFINITIONS(${READLINE_DEFINES})
 ADD_DEFINITIONS(${SSL_DEFINES})
-MYSQL_ADD_EXECUTABLE(mysql completion_hash.cc mysql.cc readline.cc sql_string.cc)
-TARGET_LINK_LIBRARIES(mysql mysqlclient)
+IF (DISABLE_SHARED)
+  MYSQL_ADD_EXECUTABLE(mysql completion_hash.cc mysql.cc readline.cc sql_string.cc)
+  SET(LIBS mysqlclient)
+ELSE(DISABLE_SHARED)
+  SET(GET_PASSWORD_SRC ../libmysql/get_password.c)
+  MYSQL_ADD_EXECUTABLE(mysql completion_hash.cc mysql.cc readline.cc sql_string.cc ${GET_PASSWORD_SRC})
+  SET(LIBS libmysql strings mysys ${SSL_LIBRARIES})
+ENDIF(DISABLE_SHARED)
+TARGET_LINK_LIBRARIES(mysql ${LIBS})
 IF(UNIX)
   TARGET_LINK_LIBRARIES(mysql ${READLINE_LIBRARY})
 ENDIF(UNIX)
 
-MYSQL_ADD_EXECUTABLE(mysqltest mysqltest.cc COMPONENT Test)
+MYSQL_ADD_EXECUTABLE(mysqltest mysqltest.cc ${GET_PASSWORD_SRC} COMPONENT Test)
 SET_SOURCE_FILES_PROPERTIES(mysqltest.cc PROPERTIES COMPILE_FLAGS "-DTHREADS")
-TARGET_LINK_LIBRARIES(mysqltest mysqlclient regex)
-
-
-MYSQL_ADD_EXECUTABLE(mysqlcheck mysqlcheck.c)
-TARGET_LINK_LIBRARIES(mysqlcheck mysqlclient)
-
-MYSQL_ADD_EXECUTABLE(mysqldump mysqldump.c ../sql-common/my_user.c)
-TARGET_LINK_LIBRARIES(mysqldump mysqlclient)
-
-MYSQL_ADD_EXECUTABLE(mysqlimport mysqlimport.c)
-TARGET_LINK_LIBRARIES(mysqlimport mysqlclient)
-
-MYSQL_ADD_EXECUTABLE(mysql_upgrade mysql_upgrade.c)
-TARGET_LINK_LIBRARIES(mysql_upgrade mysqlclient)
+TARGET_LINK_LIBRARIES(mysqltest ${LIBS} regex)
+
+
+MYSQL_ADD_EXECUTABLE(mysqlcheck mysqlcheck.c ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysqlcheck ${LIBS})
+
+MYSQL_ADD_EXECUTABLE(mysqldump mysqldump.c ../sql-common/my_user.c ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysqldump ${LIBS})
+
+MYSQL_ADD_EXECUTABLE(mysqlimport mysqlimport.c ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysqlimport ${LIBS})
+
+MYSQL_ADD_EXECUTABLE(mysql_upgrade mysql_upgrade.c ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysql_upgrade ${LIBS})
 ADD_DEPENDENCIES(mysql_upgrade GenFixPrivs)
 
-MYSQL_ADD_EXECUTABLE(mysqlshow mysqlshow.c)
-TARGET_LINK_LIBRARIES(mysqlshow mysqlclient)
+MYSQL_ADD_EXECUTABLE(mysqlshow mysqlshow.c ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysqlshow ${LIBS})
 
 MYSQL_ADD_EXECUTABLE(mysql_plugin mysql_plugin.c)
-TARGET_LINK_LIBRARIES(mysql_plugin mysqlclient)
-
-MYSQL_ADD_EXECUTABLE(mysqlbinlog mysqlbinlog.cc)
-TARGET_LINK_LIBRARIES(mysqlbinlog mysqlclient)
-
-MYSQL_ADD_EXECUTABLE(mysqladmin mysqladmin.cc)
-TARGET_LINK_LIBRARIES(mysqladmin mysqlclient)
-
-MYSQL_ADD_EXECUTABLE(mysqlslap mysqlslap.c)
+TARGET_LINK_LIBRARIES(mysql_plugin ${LIBS})
+
+MYSQL_ADD_EXECUTABLE(mysqlbinlog mysqlbinlog.cc ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysqlbinlog ${LIBS})
+
+MYSQL_ADD_EXECUTABLE(mysqladmin mysqladmin.cc ${GET_PASSWORD_SRC})
+TARGET_LINK_LIBRARIES(mysqladmin ${LIBS})
+
+MYSQL_ADD_EXECUTABLE(mysqlslap mysqlslap.c ${GET_PASSWORD_SRC})
 SET_SOURCE_FILES_PROPERTIES(mysqlslap.c PROPERTIES COMPILE_FLAGS "-DTHREADS")
-TARGET_LINK_LIBRARIES(mysqlslap mysqlclient)
+TARGET_LINK_LIBRARIES(mysqlslap ${LIBS})
 
 # "WIN32" also covers 64 bit. "echo" is used in some files below "mysql-test/".
 IF(WIN32)

